Src/Newtonsoft.Json/JsonConverter.cs

```diff
@@ -96,4 +97,75 @@ public virtual bool CanWrite
             get { return true; }
         }
     }
+
+    /// <summary>
+    /// Converts an object to and from JSON.
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
+    public abstract class JsonConverter<T> : JsonConverter
+    {
+        /// <summary>
+        /// Writes the JSON representation of the object.
+        /// </summary>
+        /// <param name="writer">The <see cref="JsonWriter"/> to write to.</param>
+        /// <param name="value">The value.</param>
+        /// <param name="serializer">The calling serializer.</param>
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
```

Comment:
I've never used sealed override but I expect it would be a breaking change for anyone who has inherited and overrided a sealed method from a converter whose base type was changed to the generic converter.